gatsby-node.js

```diff
@@ -5,6 +5,7 @@ const locales = require('./src/constants/locales');
 const DOC_LANG_FOLDERS = ['/en/', '/zh-CN/'];
 // const benchmarksMenu = require('./benchmark-menu');
 const express = require('express');
+const HTMLParser = require('node-html-parser');
```

Comment:
https://github.com/zilliztech/milvus.io.builder please update the builder once you introduce new package